lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -72,9 +72,6 @@ PrintOptions SymbolGraph::getDeclarationFragmentsPrintOptions() const {
 
   llvm::StringMap<AnyAttrKind> ExcludeAttrs;
 
-#define DECL_ATTR(SPELLING, CLASS, OPTIONS, CODE) \
-  if (StringRef(#SPELLING).startswith("_")) \
-    ExcludeAttrs.insert(std::make_pair("DAK_" #CLASS, DAK_##CLASS));
```

Comment:
I'm assuming printing already filters out `UserInaccessible` attributes, which makes this change a little *too* neat. Nice find!

Reply:
Yeah, they did 😅

Reply:
I did add `UserInaccessible` to more attributes in swift-syntax before this patch, but the only one that would have mattered was `_const`.